tests/core/gui/test_workspaces_reencrypt.py

```diff
@@ -121,17 +133,18 @@ def _reencrypt_button_displayed():
 
 @pytest.mark.gui
 @pytest.mark.trio
+@pytest.mark.flaky(reruns=3)
```

Comment:
Why is it flacky now ?

Reply:
:shrug: dunno but It can fail sometime and slow me down that why I marked it flaky